addons/frontend/index.html

```diff
@@ -8,7 +8,24 @@
     <script type="module" src="/src/svg.ts"></script>
   </head>
 
-  <body style="padding: 0; margin: 0;">
-    <div id="imageContainer"></div>
+  <body
+    style="
+      padding: 0;
+      margin: 0;
+      background-color: var(
+        --vscode-editor-background,
+        rgb(82, 86, 89)
+      ) !important;
+    "
+  >
+    <div
+      id="imageContainer"
+      style="
+        background-color: var(
+          --vscode-editor-background,
```

Comment:
vscode-editor-background is good for dark mode. But for light theme like `Light+`, `Modern Light`, the background color is white, making the pages invisible again. Perhaps we can add a stroke around the rect?

Reply:
good catch. we can retrieve a `preferred-theme` css variable, which typical has value of `dark` or `light`. We can add some graidient border to the rects in light themes.

Reply:
I tested border, and whatever they look quite ugly. So I replace vscode-editor-background by vscode-sideBar-background (color of the explorer view). This is a great hack to assume the background of explorer view is not pure white, but I don't have one theme with that...

<img width="1280" alt="image" src="https://github.com/Enter-tainer/typst-preview/assets/35292584/e42af70e-7ced-4f18-83f2-e0ce0203827c">

Reply:
<img width="673" alt="image" src="https://github.com/Enter-tainer/typst-preview/assets/25521218/4c40067f-95b3-428e-98ac-ad64b0a9b10c">

It doesn't look great but better than none. But I don't see how pages are centered horizontally in this image. Is this included  in this PR?

I'm also thinking about what will it looks like by adding a box shadow to pages(rather than border stroke)

Reply:
You've been triggering this bug, https://github.com/Enter-tainer/typst-preview/issues/86. The div is not resized according to the window sometimes.

Centering feature has been implemented here:

```js
/// center the page and add margin
const calculatedPaddedXRough = (newWidth - pageWidth) / 2;
const calculatedPaddedX = Math.abs(calculatedPaddedXRough) < 1e-3 ? 0 : calculatedPaddedXRough;
const calculatedPaddedY = accumulatedHeight + (i == 0 ? 0 : heightMargin);
const translateAttr = `translate(${calculatedPaddedX}, ${calculatedPaddedY})`;
```

![image](https://github.com/Enter-tainer/typst-preview/assets/35292584/567a4554-2834-45df-8549-193ffa659743)

![image](https://github.com/Enter-tainer/typst-preview/assets/35292584/5942d655-3a14-41dc-9365-a0e24711af26)

Reply:
i'm sure i dont trigger that bug. I didnt even change the width of the preview panel. All I did is ctrl+wheel in the preview panel.

Reply:
Okay, you did not say that you are zooming content. Indeed, the previous adjusting algorithm is broken again when we zoom out, and fixing is not included in this PR.

Reply:
okay so the "centering content horizontally" in this pr means that when the content is not zoomed, pages with smaller width will be centered? This is a bit off of what i initially expected.